amazon-msk/cdktf/iot-ingest-and-control/zilla.mustache

```diff
@@ -0,0 +1,133 @@
+name: {{name}}
+vaults:
+  secure:
+    type: aws-secrets
+{{^cloudwatchDisabled}}
```

Comment:
Do we have an option of using structured objects for this now, like a `cloudwatch` object with fields?

Reply:
Such as `{{cloudwatch.field}}` and testing if `{{cloudwatch}}` exists as an object.